integration-tests/wallet-with-bad-code.spec.ts

```diff
@@ -3,13 +3,13 @@ import { badCode } from "./data/badCode";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test contract made from wallet API failing with bad code using: ${rpc}`, () => {
+  describe(`Test contract origination fail with bad code through wallet api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('fails with a 400 RPC response', async (done) => {
+    it('Verify that wallet.originate for a contract with 400 in code will return 400 RPC in response', async (done) => {
```

Comment:
There is no 400 in the contract code, but an instruction that doesn't exist : FAILWITH_TYPO

Reply:
fixed